src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -178,6 +180,7 @@ private enum ResponseStatus
             if (brokerOptions.MsaPassthrough)
```

Comment:
If it's MSA-PT, then msal_runtime doesn't display the account picker anyway. Please see the original work item, I added a spec there.